secrets/format.go

```diff
@@ -32,7 +32,7 @@ func (result Result) toKV(prefix string) string {
 	for key, val := range result {
 		leKey := fixEnvName(key)
 		fmt.Println(leKey)
-		resturnString += fmt.Sprintf("%s%s='%s'\n", prefix, leKey, val)
+		resturnString += fmt.Sprintf("%s%s=%s\n", prefix, leKey, val)
```

Comment:
This change will fuck up the ENV part as it uses the to kv as well.

Reply:
Alright, i'll try something else

Reply:
hmm.. if I run `GITHUB_TOKEN='1234'` in my terminal and then do a `echo $GITHUB_TOKEN` the single quotes is not printed?

Reply:
Nope, it strips the single quotes are stripped by the export command, but it uses them when you parses strings containing special chars.